app/commands_app/Dockerfile

```diff
@@ -1,11 +1,16 @@
 FROM fivestarsos/commands-base:1.0.0
 
+RUN apk add --no-cache \
```

Comment:
```
Step 7/8 : RUN pip list --no-cache-dir --outdated
 ---> Running in 8aeb03cc9315
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
```

Reply:
Fine with leaving this, but I think the better pattern is to update the commands-base image to have python2 and python3 versions and inherit from that